contracts/test/TokenHomeTests.t.sol

```diff
@@ -788,6 +784,10 @@ abstract contract TokenHomeTest is TokenTransferrerTest {
         _send(input, amount);
     }
 
+    function testGetTokenAddress() public view {
```

Comment:
I see the change to add `view` but was the move  of this and others below just alphabetizing?

Reply:
No, public view functions need to go below public functions